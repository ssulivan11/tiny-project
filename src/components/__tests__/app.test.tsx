/// <reference types="enzyme-adapter-preact-pure" />

import { h } from "preact";
import { shallow } from "enzyme";
import { App } from "../app";

test("App jest tests", () => {
  const AppComponent = shallow(<App name={"test"} />);
  expect(AppComponent).toMatchSnapshot();
});
