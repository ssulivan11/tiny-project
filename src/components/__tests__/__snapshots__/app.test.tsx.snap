// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App jest tests 1`] = `
<div
  className="container"
>
  <div
    className="banner"
  >
    <h1>
      Welcome to 
      test
      !
    </h1>
    <p>
      This is just a simple demo project messing with vite for the dev server, stupid fast, no bundling and hmr.
    </p>
    <p>
      Also utilizing parcel as the bundler.
    </p>
  </div>
</div>
`;
